desktop/apps/buy_art/routes.coffee

```diff
@@ -0,0 +1,17 @@
+{ extend } = require 'underscore'
+JSONPage = require '../../components/json_page'
+resizer = require '../../components/resizer'
+markdown = require '../../components/util/markdown'
+Items = require '../../collections/items'
+
+landing = new JSONPage name: 'buy-art/landing'
+
+@landing = (req, res, next) ->
+  landing.get()
+    .then (data) ->
+      res.render 'landing', extend {},
```

Comment:
I'm not actually sure, and couldn't find any doc either. I'm guessing it's something like [lodash's extend](https://lodash.com/docs/#assignIn), though.

@xtina-starr Do you know anything about `extend`?